docs/source/developer-guide/getting_started_as_developer.md

```diff
@@ -58,7 +58,7 @@ mamba activate simtools-dev
 pip install -e .
 ```
 
-To install the CORSIKA/sim_telarray packages, follow the installation instructions in the relevant manuals. A good guideline is also the step-by-step instructions outlined in the [CORSIKA/sim_telarray docker file](https://github.com/gammasim/simtools/blob/main/docker/Dockerfile-simtelarray).
+To install the CORSIKA/sim_telarray packages, follow the installation instructions in the relevant manuals. A good guideline is also the step-by-step instructions outlined in the [CORSIKA/sim_telarray Docker file](https://github.com/gammasim/simtools/blob/main/docker/Dockerfile-prod-opt).
```

Comment:
Is the docker file here the correct one? Shouldn't it be the corsika-simtel one?